components/content/inspira/ui/morphing-text/MorphingText.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div
+    :class="
+      cn(
+        'relative mx-auto h-16 w-full max-w-screen-md text-center font-sans text-[40pt] font-bold leading-none [filter:url(#threshold)_blur(0.6px)] md:h-24 lg:text-[6rem]',
+        props.class,
+      )
+    "
+  >
+    <span
+      ref="text1Ref"
+      :class="cn(TEXT_CLASSES)"
+    />
+    <span
+      ref="text2Ref"
+      :class="cn(TEXT_CLASSES)"
+    />
+
+    <svg
+      id="filters"
+      class="hidden size-full"
+      preserveAspectRatio="xMidYMid slice"
+    >
+      <defs>
+        <filter id="threshold">
+          <feColorMatrix
+            in="SourceGraphic"
+            type="matrix"
+            values="1 0 0 0 0
+                  0 1 0 0 0
+                  0 0 1 0 0
+                  0 0 0 255 -140"
+          />
+        </filter>
+      </defs>
+    </svg>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { cn } from "@/lib/utils";
+
+const MORPH_TIME = 1.5;
```

Comment:
I think we can use these `MORPH_TIME` and `COOL_DOWN_TIME` as props. This will make this component more customizable.